stats/opencensus/e2e_test.go

```diff
@@ -1338,6 +1338,25 @@ func (fe *fakeExporter) ExportSpan(sd *trace.SpanData) {
 	fe.seenSpans = append(fe.seenSpans, gotSI)
 }
 
+// waitForServerSpan waits until a server span appears somewhere in the span
+// list in an exporter. Returns an error if no server span found within 5
+// seconds.
```

Comment:
"...within defaultTestTimeout"?

(Isn't that 10s actually?)

Also, a style point: pass in the context instead of having this timeout anyway.  Any reasonable test (including `TestSpan`) should already have a context it's using with this deadline, and we ideally shouldn't ever be declaring new contexts in the code besides `main` and `Test` functions.